src/irmin-pack/unix/control_file.ml

```diff
@@ -109,7 +109,7 @@ module Make (Io : Io.S) = struct
     in
     {
       dict_end_poff = pl.dict_end_poff;
-      suffix_end_poff = pl.suffix_end_poff;
+      appendable_chunk_poff = pl.suffix_end_poff;
```

Comment:
I think the renaming to `appendable_chunk_poff` is leaking a bit of the logic of `chunked_file` outside of it - in particular that the last chunk is appendable. For instance, this sort of info is not available to the gc_worker (https://github.com/mirage/irmin/blob/main/src/irmin-pack/unix/gc_worker.ml#L296).  

But as you wish, of course, just wanted to point out a possible issue with the renaming.

Reply:
I agree it's a bit of a concept leak, but I'm not sure what option we have since we don't have any access to the chunked suffix here (do you have other ideas?). We have to assert some knowledge about how v3 CF values upgrade to v4 CF values. I added a comment for future readers to at least explain assert why these two values are equivalent during an upgrade.

Reply:
As for the name, it came out of a conversation with @Ngoguey42 where he proposed just naming the field what it is instead of trying to name it based on its function. Outside of this one place during upgrade, I think the name works well enough and does not really leak since it goes through the chunked suffix module's api.

Reply:
I was thinking of `chunk_poff` without the `appendable_`, but I am not insisting :) both works for me